examples/enable/tools/button_tool.py

```diff
@@ -10,7 +10,7 @@
 Undoable Move Tool
 ==================
 
-This example shows how to integrate a simple component move tool with apptools
+This example shows how to integrate a simple component move tool with pyface
 undo/redo infrastructure.
```

Comment:
This is a lie. There's no undo/redo that I can see in this file.

Reply:
Yes - it looks like a copy-paste from https://github.com/enthought/enable/blob/master/examples/enable/tools/apptools/undoable_move_tool.py

Probably the fault of 5-years-ago Corran.